autozone-poc/src/components/Home.js

```diff
@@ -0,0 +1,122 @@
+  }
+  const bgImgTopCatArea1 = {
+    'backgroundImage': `url(${BgImgTopCatArea1})`
+  }
+  const bgImgTopCatArea2 = {
+    'backgroundImage': `url(${BgImgTopCatArea2})`
+  }
+  const bgImgTopCatArea3 = {
+    'backgroundImage': `url(${BgImgTopCatArea3})`
+  }
+  const bgImgCatArea = {
+    'backgroundImage': `url(${BgImgCatArea})`
+  }
+
+    return (
+      <div>
+        <ScrollUpArrowBtn/>
+        <section className="welcome_area bg-img" style={bgImgStyle}>
+          <div className="container h-100">
+            <div className="row h-100 align-items-center">
+              <div className="col-12">
+                <div className="hero-content">
+                  <h6>asoss</h6>
+                  <h2>New Collection</h2>
+                  <Button btnText="view collection" />
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+        <div className="top_catagory_area section-padding-80 clearfix">
+          <div className="container">
+            <div className="row justify-content-center">
+              <TopCatagoryArea topCatagoryAreaText="Clothing" style={bgImgTopCatArea1} />
+              <TopCatagoryArea topCatagoryAreaText="Shoes" style={bgImgTopCatArea2} />
+              <TopCatagoryArea topCatagoryAreaText="Accessories" style={bgImgTopCatArea3} />
+            </div>
+          </div>
+        </div>
+        <div className="cta-area">
+          <div className="container">
+            <div className="row">
+              <div className="col-12">
+                <div className="cta-content background-overlay bg-img" style={bgImgCatArea}>
+                  <div className="h-100 d-flex align-items-center justify-content-end">
+                    <div className="cta--text">
+                      <h6>-60%</h6>
+                      <h2>Global Sale</h2>
+                      <Button btnText="Buy Now" />
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <section className="new_arrivals_area section-padding-80 clearfix">
+          <div className="container">
+            <div className="row">
+              <div className="col-12">
+                <div className="section-heading text-center">
+                  <h2>Popular Products</h2>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="container">
+            <div className="row">
+              <div className="col-12">
+                <div className="popular-products-slides owl-carousel owl-theme owl-loaded">
+                  <SimpleSlider />
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+        <div className="brands-area d-flex align-items-center justify-content-between">
+        <div className="single-brands-logo">
+            <img src='src/utlis/img/core-img/brand1.png' alt=""/>
```

Comment:
Why this is statically implemented

Reply:
Component data implemented has dynamically

Reply:
I dont see any update in the code